src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const TodoForm = props =>{
+    return(
+        <form onSubmit = {props.handleAddTodo}> 
+            <input
+                onChange = {props.handleChanges}
+                type = "text"
+                value = {props.value}
+                placeholder = "To Do"
+                name = 'todo'/>
+                <button>Add To Do</button>  
```

Comment:
Be careful with naming the state property as `props.value` - that can get confusing when dealing with more than one input field.